src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -128,4 +128,31 @@ partial class BoundThrowExpression
     {
         protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.Expression);
     }
+
+    internal partial class BoundMethodGroup
+    {
+        protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.ReceiverOpt);
+    }
+
+    internal partial class BoundPropertyGroup
+    {
+        protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.ReceiverOpt);
+    }
+
+    internal partial class BoundSequence
```

Comment:
Why do we need this? This bound node only shows up in lowered trees.

Reply:
hmm. they were used in some of IOperation tests. that is why I added children since otherwise, test won't pass.